### PR TITLE
Add derek's tiny laptop key

### PR DESCRIPTION
**Pull request by loansindi**

### CODE REVIEW

**Comment by hef** [comment]
you have to add yourself to the init.sls file too

**Comment by loansindi** [comment]
there's another commit now

File: ssh/init.sls [hunk 1]
```diff
@@ -12,6 +12,10 @@ dbever:
     - present
     - user: root
     - source: salt://ssh/dbever.id_rsa.pub
+dbever_oberth:
```
Comment (hef) [inline ssh/init.sls]: change this to `    ssh_auth:`

Comment (loansindi) [inline ssh/init.sls]: i think i did it

**Pull request merged**